get_global_param.c

```diff
@@ -73,8 +73,8 @@ void ProgramState::initGrid(const std::vector<cell_info_struct>& cells) {
     global_param.gridStartLon = cells[0].soil_con.lng;
     global_param.gridStepLat = INT_MAX;
     global_param.gridStepLon = INT_MAX;
-    double largestLat = cells[0].soil_con.lat;
-    double largestLon = cells[0].soil_con.lng;
+    global_param.gridEndLat = cells[0].soil_con.lat;
```

Comment:
Just to be more consistent with the naming already existing for global_parm struct members, as these have been added to that struct (i.e. they are no longer temporary local variables within ProgramState::initGrid) because they need to be used in the new ProgramState::initCellMask() function.